src/styles/blocks/gallery.scss

```diff
@@ -0,0 +1,90 @@
+.gallery {
+
+  padding-bottom: 100px;
+
+  @include onDesktop {
+    display: none;
+  }
+
+  &__content {
+    padding-top: 40px;
+    display: flex;
+    flex-wrap: nowrap;
+    gap: 16px;
+    overflow: scroll;
+    scrollbar-width: none;
```

Comment:
if you want to remove scroll you need to add button in footer to scroll up

Reply:
I don't understand what you mean.  This code is for a specific block, not the whole page, why do I need to add a button to the footer? I want this particular element to have an overflow scroll(horizontal) but without a scrollbar.